tests/testthat/helper-write_parameter_file.R

```diff
@@ -6,6 +6,7 @@ write_sample_parameters_file <- function(value,
                                          parameter,
                                          start_date,
                                          end_date) {
+  Sys.sleep(0.05)
```

Comment:
I should have flagged this change as unrelated (and probably shouldn't have done it here). A single test involving this helper was a little flaky. I think it was because this helper was writing the file before `withr` could set up the temporary directory. This change has helped fix the test.